designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,57 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+Removing the tab ‘Virtual Machine Templates’ from the side navigation and combining it with ‘Virtual Machines’, so both are located under ‘Virtualization’.
+This PR will focus on how users can create a VM when they choose a to create a VM from a Template.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual Machine Templates Tab where they can see the full list of templates.
+From this point, we suggest 2 options to create a VM:
+
+- Create template via an action from the kebab menu
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+- Create template via an action from the kebab menu and also allow an action link
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
+
+Both options will open a modal where the user will need to provide a name for the VM they want to create.
+Users can click the button external link ‘View Template details’ to view the Template details page, so they don't exit the modal.
+Clicking the secondary button ‘Edit and Create’ will allow them to edit the template before they create the VM.
+
+![ModalToProvideName](img/NoMissingInfo.png)
+
+In case a user selects a template that has required info missing, they will be taken to the wizard to edit the details and then create the VM. An inline alert will be shown and indicate via a clickable link which info is missing and where it is located so by clicking it the user can quickly complete the required info.
+
+![Back to wizard to complete info](img/Step-1-basic-template-3.png)
```

Comment:
looks great! 
Nit: It will likely get placed inside the content area of the wizard so we'd have padding above the alert.
One question too: We should think about if those should be links if they are already on the page, or maybe it will scroll them to the field?

Reply:
Added padding at the top.
About the links, I was thinking they would take directly to the field. WDYT?

Reply:
+1 ya I think it depends too b/c they might need to take you to another step and into a modal. Those are more implementation details I guess. I've also added the design here for import
https://docs.google.com/document/d/1vSYmzlMOmyD7_Zp4_uGZWCKuhOSPV_pd4vWJ9KxmcZk/edit#

Reply:
I changed it to danger b/c the fields are red already.

Reply:
> I changed it to danger b/c the fields are red already.

@matthewcarleton Do you think that all the inline alerts should be red? Isn't that too stressful?

Reply:
ya I'm not sure. I think this would be a good one to get others opinions on as well. It could also be just an info. I guess the warning didn't feel like it lined up well because the fields have the error state.

Reply:
it could also be a case for a warning state on the fields which PF doesn't support but there is an issue for it.
https://github.com/patternfly/patternfly-next/issues/2598